app/src/main/java/rick_and_morty/data/repository/CharacterRepository.kt

```diff
@@ -14,7 +14,7 @@ class CharacterRepository @Inject constructor(
     }
 
 
-/*    suspend fun getRickAndMortyInfo(page: Int): List<InfoDto> {
-        return characterApiService.fetchAllCharacters(page).info!!
-    }*/
+    suspend fun getCharacterDetails(id: Int): CharacterResultsDto {
+        return rickAndMortyApiRemoteDataSource.fetchCharacterDetailsData(id)
```

Comment:
nit: use expression body syntax: https://kotlinlang.org/docs/coding-conventions.html#functions

Reply:
Is it new syntaxing rule? @LordRaydenMK

Reply:
Done